Assets/Scripts/Player.cs

```diff
@@ -58,7 +67,8 @@ private void Awake()
         DontDestroyOnLoad(this);
 
 
-
+        audioSource = GetComponent<AudioSource>(); //required
+        audioSource.Play(); //wont play one shots if play isnt called for some reason
```

Comment:
why playing on start? if this is footsteps we won't want them making sound if a player is standing still.